hips/draw/simple.py

```diff
@@ -30,13 +32,21 @@ def draw_sky_image(geometry: WCSGeometry, tiles: Generator[HipsTile, Any, Any],
         A list of HipsTile
     hips_survey : `~hips.HipsSurveyProperties`
         HiPS survey properties
+    tile_format : `str`
```

Comment:
@cdeil I don't think I can use that. The properties file could contain multiple formats such as 'jpg fits'.